oaknational/Oak-Web-Application
# fix: arialabel to have size in mb rather than bytes

## Description

- fixes aria label of downloads

## Issue(s)

Fixes #797 

## How to test

1. Go to {cloud link}
2. screenreader
3. You should read "0.3 mebabytes" in the download link

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/192569550-658d95de-9a8f-43fa-812e-cff27c89f3aa.png)


## Checklist
